wahlordnung.md

```diff
@@ -112,14 +114,29 @@ Der Wahlausschuss gibt die Ergebnisse der Wahl per E-Mail an die Fachschaft beka
 (2) Über Zulässigkeit und Folgen der Anfechtung entscheidet der Wahlausschuss.
 
 
-## § 13 Wahlordnung der Universität Potsdam
+## § 13 Rücktritte und Nachrückverfahren
+
+(1) Jedes Mitglied eines Gremiums kann aus triftigen Gründen von einem gewählten Amt zurücktreten. Der Rücktritt muss zusammen mit einer Begründung dem Fachschaftsrat in Schriftform vorgelegt werden.
+
+(2) Ein Ausscheiden aus der Fachschaft bewirkt zwingend einen sofortigen Rücktritt der Person aus allen Gremien. Dies ist dem Fachschaftsrat unverzüglich mitzuteilen.
+
+(3) Verliert eine Person das passive Wahlrecht, bleibt sie in ihrem Amt, solange sie weiterhin Mitglied der Fachschaft ist.
+
+(4) Rücktritte im Fachschaftsrat werden gemäß der Geschäftsordnung des Fachschaftsrates IT-Systems Engineering gehandhabt.
```

Comment:
Sollte ein Rücktritt beinhalten, dass man auch das Amt in der Studienkommission bzw. dem Prüfungsausschuss verliert, sollte das hier auch noch angemerkt werden.

`Ein solcher Rücktritt bedeutet den gleichzeitigen Rücktritt aus den vom Fachschaftsrat verliehenden Ämtern in der Studienkommission und dem Prüfungsausschuss, welche neu besetzt werden müssen.`